udata/core/site/views.py

```diff
@@ -96,6 +96,8 @@ def home():
     }
     processor = theme.current.get_processor('home')
     context = processor(context)
+    context['banner_activated'] = current_app.config['BANNER_ACTIVATED']
```

Comment:
I'm not sure you need this. Did you try to using global `config` context var? It should be available in the templates, cf https://github.com/opendatateam/udata/blob/master/udata/templates/apidoc.html#L5 for example.

Reply:
changed that and [adapted the gouvfr theme for it](https://github.com/etalab/udata-gouvfr/pull/480/files#diff-60ca9eea7a34ed3b3dba3f52f6d46227), thx for the tip 👍